### PR TITLE
feat(cheatcodes): Add `vm.prevrandao` to `Vm.sol`

### PR DESCRIPTION
**Pull request by Evalir**
Follow up to https://github.com/foundry-rs/foundry/pull/4856

### CODE REVIEW

File: src/Vm.sol [hunk 1]
```diff
@@ -352,6 +352,8 @@ interface Vm is VmSafe {
     function fee(uint256 newBasefee) external;
     // Sets block.difficulty
     function difficulty(uint256 newDifficulty) external;
+    // Sets block.prevrandao
+    function prevrandao(bytes32 newPrevrandao) external;
```
Comment (mds1) [inline src/Vm.sol:356]: Are these aliases for each other or will they revert depending on the EVM version you have selected in `foundry.toml`? If the latter, might be nice to document that in the comments here

Comment (Evalir) [reply src/Vm.sol:356]: Ah, actually, that's a great point—these will fail silently depending on which EVM version you have. They've not been configured/changed to revert (and REVM does not fail either). We should probably update these to have this behavior

Comment (mds1) [reply src/Vm.sol:356]: Sounds good, I'll hold off on merging this in the meantime then if you think that change will be ready soon

Comment (Evalir) [reply src/Vm.sol:356]: Following up with https://github.com/foundry-rs/foundry/pull/4904

Comment (mds1) [reply src/Vm.sol:356]: With https://github.com/foundry-rs/foundry/pull/4904 merged, @Evalir do you mind pushing a commit to tweak these comments so they're a bit more self-documenting?

**Comment by Evalir** [comment]
Ah shoot closed this accidentally by updating my fork. Will reopen

**Pull request closed without merging**